helpers/interactions/commands/createApplicationCommand.ts

```diff
@@ -89,7 +89,7 @@ export interface CreateContextApplicationCommand {
 }
 
 export function isContextApplicationCommand(
-  cmd: CreateContextApplicationCommand | CreateApplicationCommand,
-): cmd is CreateContextApplicationCommand {
+  cmd: Partial<CreateContextApplicationCommand> | Partial<CreateApplicationCommand>,
+): cmd is Partial<CreateContextApplicationCommand> {
```

Comment:
I do not like this change

Reply:
@itohatweb Why? The Docs stated that **All fields are optional**, but any fields provided will entirely overwrite the existing values of those fields.

Reply:
I feel like it's more the name of the function itself which is wrong here.

Reply:
How should I fix this?

Reply:
resolved in discord. 
> just ignore
> let this be
> what u have rn
> let this be how it is now
> yes
> like i said
> leave this